primitives/AccountId.ts

```diff
@@ -0,0 +1,2 @@
+// Any need to encode size of 32 bytes?
+export class AccountId extends Uint8Array {}
```

Comment:
This should be a branded type, not a real run-time class. A class implies the validity of e.g. `instanceof` checks, which is not the case here.